app/src/main/java/org/oppia/android/app/shim/ViewBindingShimImpl.kt

```diff
@@ -38,17 +37,18 @@ class ViewBindingShimImpl @Inject constructor() : ViewBindingShim {
     parent: ViewGroup,
     attachToParent: Boolean
   ): View {
-    return PromotedStoryCardBinding.inflate(
+    return org.oppia.android.databinding.PromotedCarouselItemCardBinding.inflate(
```

Comment:
Can you put this in the imports? Here and below.
`
import org.oppia.android.databinding.PromotedCarouselItemCardBinding`

Looking at the errors:

`ERROR: /home/runner/work/oppia-android/oppia-android/app/BUILD.bazel:616:19: KotlinCompile //app:app_kt { kt: 255, java: 0, srcjars: 0 } for armeabi-v7a failed: (Exit 1): builder failed: error executing command bazel-out/host/bin/external/io_bazel_rules_kotlin/src/main/kotlin/builder '--flagfile=bazel-out/android-armeabi-v7a-fastbuild/bin/app/app_kt-kt-java.jar-0.params'
bazel-out/android-armeabi-v7a-fastbuild/bin/app/src/main/java/org/oppia/android/app/shim/ViewBindingShimImpl_updated.kt:40:30: error: unresolved reference: databinding
    return org.oppia.android.databinding.PromotedCarouselItemCardBinding.inflate(`

Reply:
@fsharpasharp Done. PTAL